### ISSUE TITLE
Add C DSA concept implementation.

### ISSUE THREAD

**Issue by suryapratapsinghsuryavanshi** [suryapratapsinghsuryavanshi/DS-Independent#2]
Refer to the contribution guidelines and make your contribution.
Topics - All the topics are accepted under DSA.
If you have any queries please contact Us.

Email: _dev@suryapratap.ml_

**Comment by FaizFk** [comment]
Hi can you assign this to me. I will add binary search algorithm and linear Search Algorithm. If you want something else also feel free to recommend.

**Comment by suryapratapsinghsuryavanshi** [comment]
> @suryapratapsinghsuryavanshi Hi can you assign this to me. I will add binary search algorithm and linear Search Algorithm. If you want something else also feel free to recommend.

@FaizFk 
This is an open issue for everyone to make your contribution I can absolutely merge your PR if all the things work properly.

**Comment by Shivam7679** [comment]
Hi! can I add a  DSA related to stack implementation? @suryapratapsinghsuryavanshi

**Comment by suryapratapsinghsuryavanshi** [comment]
> Hi! can I add a DSA related to stack implementation? @suryapratapsinghsuryavanshi

Yes @Shivam7679 , you are free to contribute any DSA related concepts implementation.

**Comment by Shivam7679** [comment]
Thank you.
